manim/mobject/coordinate_systems.py

```diff
@@ -1355,6 +1355,35 @@ class NumberPlane(Axes):
 
     .. note:: If :attr:`x_length` or :attr:`y_length` are not defined, the plane automatically adjusts its lengths based
         on the :attr:`x_range` and :attr:`y_range` values to set the unit_size to 1.
+
+    Examples
+    --------
+
+    .. manim:: NumberPlaneExample
+        :ref_classes: NumberPlane
+        :save_last_frame:
+
+        class NumberPlaneExample(Scene):
+            def construct(self):
+                number_plane = NumberPlane(
+                x_range = [-10,10,1],
+                y_range = [-10,10,1],
+                axis_config = {
+                    "stroke_color": WHITE,
+                    "stroke_width": 2,
+                    "include_ticks": False,
+                    "include_tip": False,
+                    "line_to_number_buff": SMALL_BUFF,
+                    "label_direction": DR,
+                    "number_scale_value": 0.5,
+                },
+                background_line_style={
+                    "stroke_color": RED_D,
+                    "stroke_width": 2,
+                    "stroke_opacity": .7,
+                })
+                self.add(number_plane)
```

Comment:
Apply black formatting
```suggestion
class NumberPlaneExample(Scene):
    def construct(self):
        number_plane = NumberPlane(
            x_range=[-10, 10, 1],
            y_range=[-10, 10, 1],
            axis_config={
                "stroke_color": WHITE,
                "stroke_width": 2,
                "include_ticks": False,
                "include_tip": False,
                "line_to_number_buff": SMALL_BUFF,
                "label_direction": DR,
                "number_scale_value": 0.5,
            },
            background_line_style={
                "stroke_color": RED_D,
                "stroke_width": 2,
                "stroke_opacity": 0.7,
            },
        )
        self.add(number_plane)
```